### DISCUSSION TITLE
Ignore specific drag to snap areas

### DISCUSSION THREAD

**Discussion by MindofYano** [rxhanson/Rectangle#818] [Q&A] [answered]
I'm trying to disable everything besides the top four items in the list below. How would I do this?

defaults write com.knollsoft.Rectangle ignoredSnapAreas -int ????????????

Bit | Snap Area | Window Action
-- | -- | --
**1** | **Top** | **Maximize**
**2** | **Bottom** | **Thirds**
**3** | **Left** | **Left Half**
**4** | **Right** | **Right Half**
5 | Top Left | Top Left Corner
6 | Top Right | Top Right Corner
7 | Bottom Left | Bottom Left Corner
8 | Bottom Right | Bottom Right Corner
9 | Top Left Below Corner | Top Half
10 | Top Right Below Corner | Top Half
11 | Bottom Left Above Corner | Bottom Half
12 | Bottom Right Above Corner | Bottom Half

**Comment by rxhanson** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/13651296/168500609-e84cb8f1-0cc6-4e00-8093-8f4b2f12e057.png)

`defaults write com.knollsoft.Rectangle ignoredSnapAreas -int 4080`

then restart the app

**Reply by MindofYano** [reply]
Thanks for the prompt reply! It worked.
